ledger/ledger-api-tests/tool/BUILD.bazel

```diff
@@ -99,7 +99,7 @@ conformance_test(
         "--crt $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:server.crt))",
         "--cacrt $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:ca.crt))",
         "--pem $$(rlocation $$TEST_WORKSPACE/$(rootpath //ledger/test-common/test-certificates:server.pem.enc))",
-        "--tls-secrets-url https://raw.githubusercontent.com/digital-asset/daml/main/ledger/test-common/files/server-pem-decryption-parameters.json",
+        "--tls-secrets-url https://raw.githubusercontent.com/digital-asset/daml/c6e1ace2b700acd6a3fa97e8366c2d74486a91f4/ledger/test-common/files/server-pem-decryption-parameters.json",
```

Comment:
This is the last commit where the file existed. [The file has never been changed while it existed](https://github.com/digital-asset/daml/commits/c6e1ace2b700acd6a3fa97e8366c2d74486a91f4/ledger/test-common/files/server-pem-decryption-parameters.json).